docs/source/contributing/ci.adoc

```diff
@@ -39,6 +39,8 @@ Thus it is used to run integration tests in addition to unit tests.
 The artifacts of a successful master build can be found at link:http://artifacts.ci.centos.org/minishift/minishift/master/[artifacts.ci.centos.org/minishift/minishift/master/<BUILD_ID>].
 The artifacts of a successful pull request build can be found at link:http://artifacts.ci.centos.org/minishift/minishift/pr/[artifacts.ci.centos.org/minishift/minishift/pr/<PR_ID>].
 
+As part of master build, {project} documentation which is compiled from the _master_ branch and integrated with OpenShift documentation is hosted at link:http://artifacts.ci.centos.org/minishift/minishift/docs/master/openshift-origin/latest/minishift/index.html[artifacts.ci.centos.org/minishift/minishift/docs/master/openshift-origin/latest/minishift/].
```

Comment:
👍

Reply:
while on it, add the comment about Circle CI doing the commit message validation. I think there is a dedicated issue, so you could add it as a separate commit. While we are on it so to say

Reply:
@hferentschik This is being handled in #1800. :)

Reply:
fair enough